classes/Language.php

```diff
@@ -1089,9 +1089,10 @@ public static function downloadXLFLanguagePack($locale, &$errors = array(), $typ
         if (!is_writable(dirname($file))) {
             // @todo Throw exception
             $errors[] = Context::getContext()->getTranslator()->trans('Server does not have permissions for writing.', array(), 'Admin.International.Notification').' ('.$file.')';
+        } elseif ($content = Tools::file_get_contents($url)) {
+            file_put_contents($file, $content);
         } else {
-            $fs = new Filesystem();
-            $fs->copy($url, $file, true);
+            $errors[] = Context::getContext()->getTranslator()->trans('Language pack unavailable %s', array($url), 'Admin.International.Notification');
```

Comment:
Crowdin still has the string _Language pack unavailable._ under the Admin.International.Notification domains.

Reply:
Thanks for checking @LouiseBonnard ! 👍

Reply:
If we remove the url, the error is less verbose :(